static-assets/scripts/resources/locale-en.json

```diff
@@ -216,7 +216,8 @@
       "SITE_ID": "Site",
       "SITE_NAME": "Site name",
       "PRIMARY_TARGET_VALUE": "Primary Target Value",
-      "TARGET": "Target",
+      "TARGET": "Target Value",
```

Comment:
Why was this translation changed? (target)
The rest of languages stayed the same.